week-1/Homework/mandatory/4-tax.js

```diff
@@ -5,7 +5,13 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(number) {
+  let charge = 0.2;
+  let percent = number * charge;
+  let total = percent + number;
+  return total;
+}
```

Comment:
Altom, you could have refactorized this function, no need for the let charge variable. your variables and parameter should have meaning based on you are doing.